### PR TITLE
Added Cobblestone minion Tier XII, Added more Stats, Added Coal Minion

### PR DESCRIPTION
**Pull request by CodeSyntaxCreator**

### CODE REVIEW

File: generic/src/main/java/net/swofty/types/generic/item/items/minion/CoalMinion.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
+package net.swofty.types.generic.item.items.minion;
+
+import net.swofty.types.generic.item.ItemType;
+import net.swofty.types.generic.item.impl.CustomSkyBlockItem;
+import net.swofty.types.generic.item.impl.Minion;
+import net.swofty.types.generic.minion.MinionRegistry;
+import net.swofty.types.generic.user.statistics.ItemStatistics;
+
+public class COALMinion implements CustomSkyBlockItem, Minion {
```
Comment (Swofty-Developments) [inline generic/src/main/java/net/swofty/types/generic/item/items/minion/CoalMinion.java:9]: Class name is different to file name, this causes the compilation issue

File: generic/src/main/java/net/swofty/types/generic/minion/minions/MinionCoal.java [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+    @Override
+    public Material getHeldItem() {
+        return Material.IRON_PICKAXE;
+    }
+
+    @Override
+    public List<MinionExpectations> getExpectations() {
+        return List.of(
+                new MinionExpectations(-1, Block.Coal, Block.AIR)
```
Comment (Swofty-Developments) [inline generic/src/main/java/net/swofty/types/generic/minion/minions/MinionCoal.java:59]: What is this Block.Coal?

**Review by Swofty-Developments — CHANGES_REQUESTED** [review]

**Comment by Swofty-Developments** [comment]
[Compilation errors are still present](https://github.com/Swofty-Developments/HypixelSkyBlock/actions/runs/8097576469/job/22129214307?pr=80)

**Comment by CodeSyntaxCreator** [comment]
oh yeah i forgot the import should be fixed now

**Comment by CodeSyntaxCreator** [comment]
All checks passed accept the pull request?

**Comment by Swofty-Developments** [comment]
Thanks for the push!

**Pull request merged**